<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNewtab">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionCreateTab"/>
   </widget>
   <widget class="QMenu" name="menuDocks">
    <property name="title">
     <string>Docks</string>
    </property>
   </widget>
   <addaction name="menuNewtab"/>
   <addaction name="menuDocks"/>
  </widget>
  <action name="actionCreateTab">
   <property name="text">
    <string>Create Tab</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
